saleor/graphql/product/mutations/products.py

```diff
@@ -294,7 +296,7 @@ class ProductVariantInput(graphene.InputObjectType):
     quantity = graphene.Int(
         description='The total quantity of this variant available for sale.')
     track_inventory = graphene.Boolean(
-        required=True,
+        required=False,
```

Comment:
I intentionally set `required=true` on all boolean fields when I was integrating the API with the new dashboard. There was a problem with null values being passed into model boolean fields which resulted in a crash at the database level. The issue is to yet be investigated, but at that moment that was the quick solution that I found.